src/frontends/paddle/src/op/one_hot_v2.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs one_hot_v2(const NodeContext& node) {
+    auto data = node.get_input("X");
+    Output<Node> depth;
+    if (node.has_input("depth_tensor")) {
+        auto depth_value = node.get_input("depth_tensor");
+        depth = std::make_shared<default_opset::Squeeze>(depth_value);
+    } else {
+        auto depth_value = node.get_attribute<int>("depth");
```

Comment:
Hi can we leave it as ```const```, and also set a default value for the attributes

Reply:
1.const had been added
2.input without num_class is not permitted in paddle, so I think it is presumably not needful to set a default value. If it is necessary, the default number of classes could be set as one greater than the largest class value in the input tensor, according to the [pytorch implement](https://pytorch.org/docs/stable/generated/torch.nn.functional.one_hot.html?highlight=one_hot#torch.nn.functional.one_hot).